docs/source/getting_started.rst

```diff
@@ -181,21 +181,44 @@ How to develop a new reader
 
 To create a new reader, you should:
```

Comment:
This is not the purpose of this PR but the  "Getting Started" is oriented towards contributions and not towards our end users. I'm wondering if it is a good thing. Maybe we should put this section in a "Contributing" section and create a real "Getting started" guide with info on installation and 1 or 2 simple commands.

Reply:
Good point! I'll create an issue for this matter.